Content.Server/Wires/WireLayout.cs

```diff
@@ -0,0 +1,31 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.Wires;
+
+// WireLayout prototype.
+//
+// This is meant for ease of organizing wire sets on entities that use
+// wires. Once one of these is initialized, it should be stored in the
+// WiresSystem as a functional wire set.
+[Prototype("wireLayout")]
+public sealed class WireLayoutPrototype : IPrototype
+{
+    [IdDataFieldAttribute]
+    public string ID { get; } = default!;
+
+    [DataField("parent")]
+    public string? Parent { get; } = default!;
```

Comment:
shouldnt this use `IInheritingPrototype` and the `ParentDataField` attribute? serv3 already supports just doing inheritance for these

Reply:
oh nice, yeah i was wondering how to deal with this